### ISSUE TITLE
refactor: Remove Hardcoded Login Credentials From Webscrapers

### ISSUE THREAD

**Issue by abhiramtilakiiit** [PraneethJain/pentagon-megathon#10]
Current status:

- When building the webscapers, some temporary accounts were created to test them.
- The credentials for the temporary accounts are publically being displayed.
- This could lead to huge security issues.

Proposed changes:

TO DO:
- [ ] Create a simple script which takes the credentials as input
- [ ] Remove the current credentials and remove the temporary accounts associated with it.
- [ ] Update related Documentation

**Comment by PraneethJain** [comment]
We can replace the credentials with fake ones and direct the user to fill in the username and password that they want?

**Comment by abhiramtilakiiit** [comment]
Nah, just take standard input would be UI friendly. You can still direct the user to fill in the username and password in the documentation btw.
